### DISCUSSION TITLE
Password for Plug-Ins?

### DISCUSSION THREAD

**Discussion by Geunzai** [Onemind-Services-LLC/netbox-secrets#109] [Q&A] [answered]
Hi, I'm new to Netbox and found out netbox-secrets recently. 

It works perfectly on most situations but it seems like it doesn't support other plugins such as netbox-documents.

For instance, I typed in  PLUGINS_CONFIG = { 'netbox_secrets': {apps: ['netbox_documents.sitedocument']}} to add passwords for site documents but the secret panel was still missing.

Is there any workaround for this?

**Comment by abhi1693** [comment, marked as answer]
You'll have to ask the plugin developer of NetBox documents to include the plugin related tags in their templates

**Reply by Geunzai** [reply]
Thank you.
